### DISCUSSION TITLE
How about apply font family via env var?

### DISCUSSION THREAD

**Discussion by ryuheechul** [charmbracelet/vhs#379] [Ideas]
First time using vhs and I'm super impressed with how well this works with just few simple steps!

I have an idea to improve recording UX a bit though.

When I tried `vhs record > demo.tape`, I noticed that it doesn't know anything about the font family so I needed to include `Set FontFamily "FiraMono Nerd Font Mono"` manually after recording which is a small added step that shouldn't be a big deal.
However, in case these `.tape` files are included in git repo and expected to run in different environments, having hard coded font could be a problem since we will not know which (nerd)font is installed on each environment.

Therefore, if `vhs` read an env var something like `VHS_FONT_FAMILY` and apply this automatically instead, `*.tape` files will be more environment agnostic.

What's your thoughts?

**Comment by maaslalani** [comment]
I like this idea generally. I think it can be solved by allowing all of the settings to be provided via flags, then the environment variables can be used as well.

**Reply by ryuheechul** [reply]
yeah, flagging would be a similar solution 👍🏼
